src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java

```diff
@@ -172,9 +173,14 @@ protected void handleAudioVideoDocuments(SscsCaseData sscsCaseData) {
         validateDwpResponseDocuments(sscsCaseData, preSubmitCallbackResponse);
 
         validateDwpAudioVideoEvidence(sscsCaseData, preSubmitCallbackResponse);
+
+
```

Comment:
No need for all these blank lines. Suggest you just revert all changes to this file as they are not needed.